### DISCUSSION TITLE
Uploading SARIF file of multi-project build

### DISCUSSION THREAD

**Discussion by CommanderTvis** [detekt/detekt#5052] [Q&A] [answered]
Is there a way to upload a single SARIF report by Detekt from Gradle multi-project build?

The Github Actions' limitation is that it supports only one SARIF report per tool.

**Comment by schalkms** [comment, marked as answer]
Yes, it is. Please take a look at [this page](https://detekt.dev/docs/introduction/reporting#merging-reports) on how to setup the Gradle script.
If you have questions, please feel free to raise follow-up questions.

**Reply by CommanderTvis** [reply]
Thank you!
